src/util/nodeSelectors.ts

```diff
@@ -200,13 +200,21 @@ export function selectWithLeadingDelimiter(...delimiters: string[]) {
  */
 export function childRangeSelector(
   typesToExclude: string[] = [],
-  typesToInclude: string[] = []
+  typesToInclude: string[] = [],
+  useUnnamedStartAndEnd: boolean = false
```

Comment:
@pokey I'm not sure about this variable name. The prior behavior ignored unnamed nodes at the start or end of the range (but did include any inside the range). This optional parameter keeps the default behavior the same but allows us to include unnamed nodes if desired. Could also be broken up into a separate start and end parameter.